src/tests/test_hiv_status.py

```diff
@@ -140,9 +148,11 @@ def test_hiv_update2():
     pop.data[col.VIRAL_LOAD_GROUP] = 5  # set everyone to same viral load group
     prev_hiv_status = pop.data[col.HIV_STATUS].copy()
     hiv_module.update_HIV_status(pop)
-    expected_transmission_prob = 0.1 * 0.1  # 10% probability of HIV+ and 10% probability of transmission given viral load
+    # 10% probability of HIV+ and 10% probability of transmission given viral loa
+    expected_transmission_prob = 0.1 * 0.1
     pop.data["delta_HIV"] = pop.data[col.HIV_STATUS] ^ prev_hiv_status
-    n_new_male = sum(pop.data.loc[pop.data[col.SEX]==SexType.Male, "delta_HIV"])  # count number of new transmissions for men 
+    # count number of new transmissions for men
+    n_new_male = sum(pop.data.loc[pop.data[col.SEX] == SexType.Male, "delta_HIV"])
     exp_new_male = 45000 * expected_transmission_prob
     sig = np.sqrt(45000 * expected_transmission_prob * (1 - expected_transmission_prob))
-    print(n_new_male, exp_new_male, sig)
\ No newline at end of file
+    print(n_new_male, exp_new_male, sig)
```

Comment:
Remove when done :grin:

Reply:
This whole test was already removed in [a62a146](https://github.com/UCL/hivpy/pull/70/commits/a62a1461fab2b11dc63fba38253f8590a642f28a)